src/FSharp.Control.TaskSeq/TaskSeq.fsi

```diff
@@ -215,14 +215,55 @@ type TaskSeq =
 
     /// <summary>
     /// Combines the given task sequence of task sequences and concatenates them end-to-end, to form a
-    /// new flattened, single task sequence. Each task sequence is awaited item by item, before the next is iterated.
+    /// new flattened, single task sequence, like <paramref name="TaskSeq.collect id"/>. Each task sequence is
+    /// awaited and consumed in full, before the next one is iterated.
     /// </summary>
     ///
     /// <param name="sources">The input task-sequence-of-task-sequences.</param>
-    /// <returns>The resulting task sequence.</returns>
+    /// <returns>The resulting, flattened task sequence.</returns>
     /// <exception cref="T:ArgumentNullException">Thrown when the input task sequence of task sequences is null.</exception>
     static member concat: sources: TaskSeq<#TaskSeq<'T>> -> TaskSeq<'T>
 
+    /// <summary>
+    /// Combines the given task sequence of sequences and concatenates them end-to-end, to form a
+    /// new flattened, single task sequence.
+    /// </summary>
+    ///
+    /// <param name="sources">The input task sequence of sequences.</param>
+    /// <returns>The resulting, flattened task sequence.</returns>
+    /// <exception cref="T:ArgumentNullException">Thrown when the input task sequence of task sequences is null.</exception>
```

Comment:
I guess Seq doesnt either but I'd like to see mention of NRE happening if there is a `null` in the input
esp for arrays.
But then LINQ SelectMany also does not document it and behaves the same way 
So I guess that's case closed based on prior art
(I can recall writing test snippets in 2007 because of this not being written anywhere I was aware of)

But kudos for having the test in the first instance so it's all good I guess